akka-cluster/src/multi-jvm/scala/akka/cluster/RemoteFeaturesWithWatcherClusterWatcheeNoClusterSpec.scala

```diff
@@ -0,0 +1,198 @@
+      role: RoleName,
+      path: String,
+      actorName: String,
+      within: FiniteDuration = 10.seconds): ActorRef = {
+    system.actorSelection(node(role) / path / actorName) ! Identify(actorName)
+    val id = expectMsgType[ActorIdentity](within)
+    assert(id.ref.isDefined, s"Unable to Identify actor [$actorName] on node [$role].")
+    id.ref.get
+  }
+
+  private val provider = RARP(system).provider
+
+  s"Remoting with UseUnsafeRemoteFeaturesWithoutCluster enabled=$useUnsafe, " +
+  "watcher system using `cluster`, but watchee system using `remote`" must {
+
+    val send = if (system.settings.HasCluster || (!system.settings.HasCluster && useUnsafe)) "send" else "not send"
+
+    s"$send `Watch`/`Unwatch`/`Terminate` when watching from cluster to non-cluster remoting watchee" in {
+      runOn(remoting) {
+        system.actorOf(Props(classOf[Listener], probe.ref), "watchee")
+        enterBarrier("watchee-created")
+        enterBarrier("watcher-created")
+      }
+
+      runOn(clustered) {
+        enterBarrier("watchee-created")
+        val watcher = system.actorOf(Props(classOf[Listener], probe.ref), "watcher")
+        enterBarrier("watcher-created")
+
+        val watchee = identify(remoting, "watchee")
+        probe.send(watcher, WatchIt(watchee))
+        probe.expectMsg(1.second, Ack)
+        provider.remoteWatcher.get ! Stats
+        awaitAssert(expectMsgType[Stats].watchingRefs == Set((watchee, watcher)), 2.seconds)
+      }
+      enterBarrier("cluster-watching-remote")
+
+      runOn(remoting) {
+        system.stop(identify(remoting, "watchee"))
+        enterBarrier("watchee-stopped")
+      }
+
+      runOn(clustered) {
+        enterBarrier("watchee-stopped")
+        probe.expectMsgType[WrappedTerminated](2.seconds)
+      }
+    }
+
+    s"$send `Watch`/`Unwatch`/`Terminate` when watching from non-cluster remoting to cluster watchee" in {
+      runOn(clustered) {
+        system.actorOf(Props(classOf[Listener], probe.ref), "watchee2")
+        enterBarrier("watchee2-created")
+        enterBarrier("watcher2-created")
+      }
+
+      runOn(remoting) {
+        enterBarrier("watchee2-created")
+        val watchee = identify(clustered, "watchee2")
+
+        val watcher = system.actorOf(Props(classOf[Listener], probe.ref), "watcher2")
+        enterBarrier("watcher2-created")
+
+        probe.send(watcher, WatchIt(watchee))
+        probe.expectMsg(1.second, Ack)
+
+        if (useUnsafe) {
+          provider.remoteWatcher.get ! Stats
+          awaitAssert(expectMsgType[Stats].watchingRefs == Set((watchee, watcher)), 2.seconds)
+        }
+      }
+
+      runOn(clustered) {
+        system.stop(identify(clustered, "watchee2"))
+        enterBarrier("watchee2-stopped")
+      }
+
+      runOn(remoting) {
+        enterBarrier("watchee2-stopped")
+        if (useUnsafe)
+          probe.expectMsgType[WrappedTerminated](2.seconds)
+        else
+          probe.expectNoMessage(2.seconds)
```

Comment:
this is correct, and it works here because from `remoting` the RemoteWatcher isn't even created and we stop the messages already in the intercept